templates/new_service/.github/workflows/build-and-deploy.yml

```diff
@@ -0,0 +1,94 @@
+name: Build and Deploy
+concurrency: build_and_deploy_${{ github.ref_name }}
+
+on:
+  push:
+   branches:
+    - main
+  pull_request:
+    branches:
+    - main
+    types: [opened, reopened, synchronize, labeled]
+
+permissions:
```

Comment:
The permissions are included in [the default permissions](https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token). Do we need them here?

Reply:
removed